src/vpn-api/utils/openvpn.ts

```diff
@@ -185,8 +185,8 @@ export class VpnManager extends EventEmitter {
 				'established',
 				'disconnect',
 			]) {
-				if (data.substr(0, eventType.length) === eventType.toUpperCase()) {
-					const eventData = data.substr(8);
+				const eventData = data.substr('>CLIENT:'.length);
```

Comment:
This should be outside of the `for` loop so that it only needs to be done once

Reply:
Typo, fixed :)